webrtc.html

```diff
@@ -399,23 +399,31 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>Let <var>p</var> be the result of transforming
+            <code>operations</code> by a fulfillment handler that runs the
```

Comment:
That's what I meant yes. How does it need to be phrased? I tried as much as possible to rely on existing language in order to keep general cleanup of this section orthogonal to this PR.